src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/utils/runners/TestRunner.java

```diff
@@ -0,0 +1,58 @@
+package com.softserveinc.ita.dkrutenko.pageobjects.utils.runners;
```

Comment:
move utils package from pageobjects package

Reply:
done

Reply:
it still there

Reply:
it didnt pushed... Dont know why... Now its fine